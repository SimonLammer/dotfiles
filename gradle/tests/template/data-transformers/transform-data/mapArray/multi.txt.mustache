map:
  {{#multiItemMapArray}}
  {{key}}: {{value}}
  {{/multiItemMapArray}}