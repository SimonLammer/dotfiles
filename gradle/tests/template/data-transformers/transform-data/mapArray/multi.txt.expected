map:
  firstKey: FirstValue
  secondKey: SecondValue
  thirdKey: ThirdValue
